static/app/views/organizationGroupDetails/quickTrace/configureDistributedTracing.tsx

```diff
@@ -84,13 +84,57 @@ class EventQuickTrace extends Component<Props, State> {
   }
 
   createDocsLink() {
-    const platform = this.props.group.project.platform ?? null;
+    const platform = this.props.project.platform ?? null;
     const docsPlatform = platform ? getDocsPlatform(platform, true) : null;
     return docsPlatform === null
       ? null // this platform does not support performance
       : `https://docs.sentry.io/platforms/${docsPlatform}/performance/`;
   }
 
+  renderActionButton(docsLink: string) {
+    const features = ['organizations:performance-view'];
+    const noFeatureMessage = t('Requires performance feature.');
```

Comment:
```suggestion
    const noFeatureMessage = t('Requires performance features.');
```

Could also use 'requires performance monitoring'